app/models/ability.rb

```diff
@@ -12,6 +12,7 @@ def initialize(user)
     can :manage, Spotlight::Resources::DorHarvester, exhibit_id: user.exhibit_roles.pluck(:resource_id)
 
     # disable spotlight functionality we don't want to expose in spotlight:
+    can :create, Spotlight::Exhibit if user.superadmin?
```

Comment:
I don't believe this line is necessary -- this ability already exists. Part of the testing I was doing, so feel free to ignore in terms of overall code quality for now.